Fix local test run with OpenCV image reading

```
[----------] 2 tests from Read_img
[ RUN      ] Read_img.can_read_image
[ WARN:0@11.977] global loadsave.cpp:248 findDecoder imread_('../image.jpg'): can't open/read file: check file path/integrity
[       OK ] Read_img.can_read_image (6 ms)
[ RUN      ] Read_img.can_save_image
[ WARN:0@11.977] global loadsave.cpp:248 findDecoder imread_('../image.jpg'): can't open/read file: check file path/integrity
/Users/arseniy/Projects/itlab/itlab_2023/test/test_read/test_read.cpp:21: Failure
Expected: imwrite(output_file_name, image); doesn't throw an exception.
  Actual: it throws cv::Exception with description "OpenCV(4.9.0-dev) /Users/arseniy/Projects/itlab/itlab_2023/3rdparty/opencv/modules/imgcodecs/src/loadsave.cpp:786: error: (-215:Assertion failed) !_img.empty() in function 'imwrite'
".

[  FAILED  ] Read_img.can_save_image (0 ms)
[----------] 2 tests from Read_img (6 ms total)

[----------] Global test environment tear-down
[==========] 44 tests from 8 test suites ran. (11952 ms total)
[  PASSED  ] 43 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] Read_img.can_save_image

 1 FAILED TEST


0% tests passed, 1 tests failed out of 1

Total Test time (real) =  12.04 sec
```

--- a/app/CMakeLists.txt
+++ b/app/CMakeLists.txt
@@ -7,7 +7,6 @@ add_custom_command(TARGET Reader POST_BUILD
         "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build/bin/${CMAKE_BUILD_TYPE}"
         "${CMAKE_BINARY_DIR}/app/${CMAKE_BUILD_TYPE}/")
 endif()
-
 file(DOWNLOAD
     "https://raw.githubusercontent.com/opencv/opencv/4.x/samples/data/lena.jpg"
     "${CMAKE_CURRENT_BINARY_DIR}/image.jpg"
@@ -15,4 +14,4 @@ file(DOWNLOAD
     STATUS status_code
     LOG log_file
 )
-add_definitions(-DIMAGE_PATH="${CMAKE_CURRENT_BINARY_DIR}/image.jpg")
+add_definitions(-DIMAGE1_PATH="${CMAKE_CURRENT_BINARY_DIR}/image.jpg")

--- a/app/CMakeLists.txt
+++ b/app/CMakeLists.txt
@@ -15,3 +15,4 @@ file(DOWNLOAD
     STATUS status_code
     LOG log_file
 )
+add_definitions(-DIMAGE_PATH="${CMAKE_CURRENT_BINARY_DIR}/image.jpg")

--- a/app/reader_img.cpp
+++ b/app/reader_img.cpp
@@ -2,6 +2,6 @@
 
 #include <iostream>
 int main() {
-  std::string imagePath = IMAGE1_PATH;
-  read(imagePath);
+  std::string image_path = IMAGE1_PATH;
+  read(image_path);
 }

--- a/app/reader_img.cpp
+++ b/app/reader_img.cpp
@@ -3,5 +3,5 @@
 #include <iostream>
 int main() {
   //const std::string imagePath = "IMAGE_PATH";
-  read((std::string)IMAGE1_PATH);
+  read(std::string(IMAGE1_PATH));
 }

--- a/app/reader_img.cpp
+++ b/app/reader_img.cpp
@@ -2,6 +2,6 @@
 
 #include <iostream>
 int main() {
-  std::string IMAGE_PATH = IMAGE_PATH;
-  read(IMAGE_PATH);
+  //const std::string imagePath = "IMAGE_PATH";
+  read((std::string)IMAGE1_PATH);
 }

--- a/app/reader_img.cpp
+++ b/app/reader_img.cpp
@@ -3,6 +3,6 @@
 #include <iostream>
 int main() {
   std::string path;
-  std::cin >> path;
+  std::cin >> IMAGE_PATH;
   read(path);
 }

--- a/app/reader_img.cpp
+++ b/app/reader_img.cpp
@@ -2,7 +2,6 @@
 
 #include <iostream>
 int main() {
-  std::string path;
   std::cin >> IMAGE_PATH;
-  read(path);
+  read(IMAGE_PATH);
 }

--- a/app/reader_img.cpp
+++ b/app/reader_img.cpp
@@ -2,6 +2,6 @@
 
 #include <iostream>
 int main() {
-  std::cin >> IMAGE_PATH;
+  std::string IMAGE_PATH = IMAGE_PATH;
   read(IMAGE_PATH);
 }

--- a/app/reader_img.cpp
+++ b/app/reader_img.cpp
@@ -2,6 +2,5 @@
 
 #include <iostream>
 int main() {
-  //const std::string imagePath = "IMAGE_PATH";
   read(std::string(IMAGE1_PATH));
 }

--- a/app/reader_img.cpp
+++ b/app/reader_img.cpp
@@ -2,5 +2,6 @@
 
 #include <iostream>
 int main() {
-  read(std::string(IMAGE1_PATH));
+  std::string imagePath = IMAGE1_PATH;
+  read(imagePath);
 }

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -28,5 +28,4 @@ file(DOWNLOAD
     STATUS status_code
     LOG log_file
 )
-# Установка пути к изображению как макрос
 add_definitions(-DIMAGE_PATH="${CMAKE_CURRENT_BINARY_DIR}/image.jpg")

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -28,3 +28,5 @@ file(DOWNLOAD
     STATUS status_code
     LOG log_file
 )
+# Установка пути к изображению как макрос
+add_definitions(-DIMAGE_PATH="${CMAKE_CURRENT_BINARY_DIR}/image.jpg")

--- a/test/test_read/test_read.cpp
+++ b/test/test_read/test_read.cpp
@@ -6,8 +6,8 @@
 using namespace cv;
 
 TEST(Read_img, can_read_image) {
-  const std::string imagePath = IMAGE_PATH;
-  ASSERT_NO_THROW(Mat image = imread(imagePath););
+  const std::string image_path = IMAGE_PATH;
+  ASSERT_NO_THROW(Mat image = imread(image_path););
 }
 TEST(Read_img, can_save_image) {
   const std::string imagePath = IMAGE_PATH;

--- a/test/test_read/test_read.cpp
+++ b/test/test_read/test_read.cpp
@@ -10,8 +10,8 @@ TEST(Read_img, can_read_image) {
   ASSERT_NO_THROW(Mat image = imread(image_path););
 }
 TEST(Read_img, can_save_image) {
-  const std::string imagePath = IMAGE_PATH;
-  Mat image = imread(imagePath);
+  const std::string image_path = IMAGE_PATH;
+  Mat image = imread(image_path);
   String output_file_name = "output_image.jpg";
   ASSERT_NO_THROW(imwrite(output_file_name, image););
 }

--- a/test/test_read/test_read.cpp
+++ b/test/test_read/test_read.cpp
@@ -5,18 +5,13 @@
 #include "read/reader_img.hpp"
 using namespace cv;
 
-std::string getImagePath() {
-  char* workspace = getenv("GITHUB_WORKSPACE");
-  if (workspace != nullptr) {
-    return std::string(workspace) + "/build/test/image.jpg";
-  }
-  return "../image.jpg";
-}
 TEST(Read_img, can_read_image) {
-  ASSERT_NO_THROW(Mat image = imread(getImagePath()););
+  const std::string imagePath = IMAGE_PATH;
+  ASSERT_NO_THROW(Mat image = imread(imagePath););
 }
 TEST(Read_img, can_save_image) {
-  Mat image = imread(getImagePath());
+  const std::string imagePath = IMAGE_PATH;
+  Mat image = imread(imagePath);
   String output_file_name = "output_image.jpg";
   ASSERT_NO_THROW(imwrite(output_file_name, image););
 }

